src/core/internal/cppdemangle.d

```diff
@@ -0,0 +1,196 @@
+/**
+* This module provide the demangle for converts mangled C++ symbols
+*
+* Copyright: Copyright © 2019, The D Language Foundation
+* License: <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
+* Authors: Ernesto Castellotti
+* Source: $(DRUNTIMESRC core/internal/_cppdemangle.d)
+*/
+
+/**
+* Demangles C++ mangled names. If it is not a C++ mangled name, it returns its argument name.
```

Comment:
Take a look at how the doc looks like. My guess is that the second sentence should be part of the description (so you need a double newline, like in the `suggestion` above).